docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,326 @@
+Microsoft PKI (MSPKI) Quick Installation Guide
+==============================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+
+This guide has been created to give a quick start to configure the Microsoft PKI with PacketFence 5.4+. This guide does not include advanced troubleshooting of EAP-TLS connections. Refer to the relevant documentation of EAP-TLS, RADIUS and OpenSSL for advanced features.
+
+Assumptions
+-----------
+
+* You have at least one server with PacketFence 5.4 or later.
+* The server already has a properly configured switch with 802.1X support.
+* The PacketFence RADIUS server is working in your environment.
+* You have an Active Directory server or another Microsoft Windows 2008 R2 server installed.
+* The PacketFence management IP will be 192.168.1.5.
+* The RADIUS shared secret is "useStrongerSecret".
+
+Installation
+------------
+
+Step 1: Install Active Directory Certificate Service (ADCS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: This will cover the configuration for Active Directory Certificate Services (ADCS) on Microsoft Windows 2008 R2. This configuration has been tested to be integreted with PacketFence. The installation of ADCS is not cover by this guide see the Microsoft white paper about it for more information (http://social.technet.microsoft.com/wiki/contents/articles/9063.network-device-enrollment-service-ndes-in-active-directory-certificate-services-ad-cs.aspx).
```

Comment:
Also important information, even if you install Network Device Enrollment Service on a 2008 R2 Enterprise you also need to have the CA on a 2008 R2 Enterprise too. (Remote CA)